### PR TITLE
fix: update typo from isWalletConnnectFactory to isWalletConnectFactory

### PR DESCRIPTION
**Pull request by liuchuzhang**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design-web3/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

## 💡 Background and solution

fix typo

### CODE REVIEW

**Comment by kiner-tang** [comment]
@liuchuzhang 感谢你的贡献，请参考这个指引为本次修改增加 changelog：https://web3.ant.design/guide/changelog-cn

File: .changeset/cuddly-mugs-compete.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@ant-design/web3-solana": major
```
Comment (thinkasany) [inline .changeset/cuddly-mugs-compete.md:2]:
```suggestion
"@ant-design/web3-solana": patch
```

Comment (liuchuzhang) [reply .changeset/cuddly-mugs-compete.md]: Thanks!

File: .changeset/cuddly-mugs-compete.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@ant-design/web3-solana": patch
+---
+
+fix typo `isWalletConnnectFactory` to `isWalletConnectFactory`
```
Comment (kiner-tang) [inline .changeset/cuddly-mugs-compete.md:5]:
```suggestion
fix: update typo from `isWalletConnnectFactory` to `isWalletConnectFactory`
```

**Review by thinkasany — APPROVED** [review]

**Pull request merged**